README.adoc

```diff
@@ -141,53 +141,53 @@ include::hotspots/src/test/java/io/openliberty/guides/testing/PersonServiceIT.1.
 //include::hotspots/src/test/java/io/openliberty/guides/testing/PersonServiceIT.2.java[]
 //----
 
-Next, we will define some basic information that informs MicroShed Testing if and how it should start our applictaion runtime, and what URL path the application will be available at:
+Next, this guide outlines some basic information that informs how MicroShed Testing starts the application runtime and at which URL path the application will be available:
 
 [role="code_command hotspot file=1", subs="quotes"]
 ----
 #Update the `PersonServiceIT` class.#
 `src/test/java/io/openliberty/guides/testing/PersonServiceIT.java`
 ----
 [role="edit_command_text"]
-Import [hotspot=importMPApp file=1]`MicroProfileApplication` class and [hotspot=importContainer file=1]`Container` annotation, create the [hotspot=mpApp file=1]`MicroProfileApplication` application, and annotate with [hotspot=container file=1]`@Container`.
+Import the [hotspot=importMPApp file=1]`MicroProfileApplication` class and the [hotspot=importContainer file=1]`Container` annotation, create the [hotspot=mpApp file=1]`MicroProfileApplication` application, and annotate the application with [hotspot=container file=1]`@Container`.
 
 PersonServiceIT.java
 [source, java, linenums, role='code_column hide_tags=copyright,importInject,importAssertNotNull,importSharedContainerConfig,sharedContainerConfig,inject,personSvc,testCreatePerson']
 ----
 include::hotspots/src/test/java/io/openliberty/guides/testing/PersonServiceIT.2.java[]
 ----
 
-The [hotspot=withAppContextRoot file=1]`withAppContextRoot(String)` method indicates what the base path of the application is. In this case, our application is deployed at http://localhost:9080/guide-microshed-testing[http://localhost:9080/guide-microshed-testing^], so the app context root is [hotspot=withAppContextRoot file=1]`/guide-microshed-testing`, which is the portion of the URL after the hostname and port.
+The [hotspot=withAppContextRoot file=1]`withAppContextRoot(String)` method indicates the base path of the application. An app context root is the portion of the URL after the hostname and port. In this case, the application is deployed at the `\http://localhost:9080/guide-microshed-testing` URL, so the app context root is [hotspot=withAppContextRoot file=1]`/guide-microshed-testing`.
```

Comment:
Instead of `An app context root` we should say `The app context root` because there is only 1 app context root

Reply:
also on this line, looks like there is a extra leading `\` char in the text:
```
the `\http://localhost:9080/guide-microshed-testing` URL
```

Reply:
the leading `\` is required, because we don't want to generate the url to be a link that return nothing.

Reply:
Agreed that we need to remove the leading `\` character. And `The app context root` works too. I was thinking that it was kind of a more general "thing" that might exist outside of this guide, but if we're just defining app context root for the scope of the guide then you're right.

Reply:
The `\` won't be shown up and the guide just won't generate the url to be a link.